packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -24,18 +51,61 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
     : transition
 }
 
+export const getSpotlightAnimation = (
+  element: HTMLDivElement,
+  targetId: string | null | undefined,
+  groupIdsInSpotlight: string[],
+  isClose: boolean,
+  lastZIndex: number | undefined
+) => {
+  const spotlightEl = document.getElementById('spotlight')
+  const isTargetInSpotlight = targetId && groupIdsInSpotlight.includes(targetId)
+  const isInSpotlight = spotlightEl && spotlightEl.contains(element)
+  const notInSpotlight = spotlightEl && !spotlightEl.contains(element)
+  const showAboveSpotlight = isInSpotlight || isTargetInSpotlight
+
+  const fadeInAnimation = `${reflectionSpotlightFadeIn.toString()} 0.5s linear 0s forwards`
```

Comment:
let's use `BezierCurve.DECELERATE` instead of linear. the different is _real_ small, but basically it means 80% of the transition is completed in the first 50% of the time.